### PR TITLE
Make UrlScan API work with updated models

### PR DESCRIPTION
**Pull request by RajuKoushik**
Signed off by - Raju Koushik g.rajukoushik@gmail.com

### CODE REVIEW

**Review by singh1114 — COMMENTED** [review]
Do you want to commit this changes after #63?

File: scanapp/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (singh1114) [inline scanapp/migrations/0001_initial.py:1]: Nothing in there?

File: scanapp/tests.py [hunk 2]
```diff
@@ -1,8 +1,12 @@
+# Create your tests here.
```
Comment (singh1114) [inline scanapp/tests.py:1]: change your tests formats.

File: scanapp/tests.py [hunk 3]
```diff
@@ -14,3 +18,25 @@ def runtests():
     test_runner = TestRunner(verbosity=1, interactive=True)
     failures = test_runner.run_tests(['your_app'])
     sys.exit(bool(failures))
+
+
+class URLScanFormTests(TestCase):
```
Comment (singh1114) [inline scanapp/tests.py:23]: Why new function?

File: scanapp/views.py [hunk 4]
```diff
@@ -155,3 +155,61 @@ def post(self, request):
                 }
             )
         )
+
+
+class URLFormViewFinal(FormView):
```
Comment (singh1114) [inline scanapp/views.py:160]: Why new function?

File: scanapp/tests.py (hunk 2 above)
Comment (RajuKoushik) [reply scanapp/tests.py:1]: Could you explain this is a bit  ?

Comment (singh1114) [reply scanapp/tests.py:1]: See the way tests are written in other PR.

Comment (RajuKoushik) [reply scanapp/tests.py:1]: Sure !

**Pull request closed without merging**